circle.yml

```diff
@@ -0,0 +1,6 @@
+machine:
+  node:
+    version: 6.9.4
+dependencies:
+  pre:
+    - npm install -g npm
```

Comment:
create a `.nvmrc` file and specify the node version there also so anyone who uses nvm can automatically switch to that node version.